gensrc/proto/starlake.proto

```diff
@@ -57,4 +60,30 @@ message TxnLogPB {
     optional OpSchemaChange op_schema_change = 5;
 }
 
+message PublishVersionRequest {
+    repeated int64 tablet_ids = 1;
+    repeated int64 txn_ids = 2;
+    optional int64 base_version = 3;
+    optional int64 publish_version = 4;
+}
+
+message PublishVersionResponse {
+    optional StatusPB status = 1;
+    repeated int64 failed_tablets = 2;
```

Comment:
In @decster new design, publish response should contain the max continuous version of published tablet.

Reply:
The lake table has a different protocol.